ui/css/itcss/components/network.scss

```diff
@@ -1,195 +0,0 @@
-@use "design-system";
-
-.network-component--disabled {
-  cursor: not-allowed;
-
-  .fa-caret-down {
-    opacity: 0;
-  }
-}
-
-.network-component.pointer {
-  border: 1px solid var(--color-border-muted);
-  border-radius: 82px;
-  padding: 6px 3px;
-  flex: 0 0 auto;
-  display: flex;
-
-  &.ethereum-network .menu-icon-circle div {
-    background-color: rgba(3, 135, 137, 0.7) !important;
-  }
-
-  &.goerli-test-network .menu-icon-circle div {
-    background-color: rgba(48, 153, 242, 0.7) !important;
-  }
-
-  &.sepolia-test-network .menu-icon-circle div {
-    background-color: rgba(207, 181, 240, 0.7) !important;
-  }
-
-  &.linea-test-network .menu-icon-circle div {
-    background-color: rgba(0, 0, 0, 0.7) !important;
-  }
-
-  &.localhost-network .menu-icon-circle div {
-    background-color: rgba(3, 135, 137, 0.7) !important;
-  }
-}
-
-.dropdown-menu-item {
-  .menu-icon-circle,
-  .menu-icon-circle--active {
-    margin: 0 14px;
-  }
-}
-
-.dropdown-menu-item .fa.delete {
-  color: var(--color-icon-default);
-  margin-right: 10px;
-  display: none;
-}
-
-.dropdown-menu-item:hover .fa.delete {
-  display: inherit;
-}
```

Comment:
Not used 

![Screenshot 2025-02-07 at 11 48 46 AM](https://github.com/user-attachments/assets/cfd8d337-5c60-4a92-b096-f03abf7b128e)